src/ucm/event/event.c

```diff
@@ -588,6 +588,17 @@ ucs_status_t ucm_test_events(int events)
 {
     int out_events;
 
+    if (events & UCM_EVENT_VM_MAPPED) {
+        events |= UCM_EVENT_MMAP | UCM_EVENT_MREMAP |
```

Comment:
Maybe create a macro aggregating these events? 
At least ucm_event_install uses the same sets

Reply:
good idea... done